ikura/shim/src/cli.rs

```diff
@@ -48,6 +48,9 @@ pub struct KeyManagementParams {
     ///
     /// This key is enabled when running ikura-node in the local development mode.
     ///
+    /// NOTE: Submitting blobs is assumed to have complete control over the signing key. In case
+    /// there is another process running with the same key, it may cause error during submission.
```

Comment:
Wouldn't it be better to be more general here? Using 'another process' might mislead the user into thinking they can only run one shim processes with the same key. However, the assumption is stronger (please correct me if I'm wrong with [this]([url](https://github.com/thrumdev/blobs/pull/270#discussion_r1512477101)) example)

Reply:
The assumption is indeed stronger, but that's precisely what I was trying to convey by "another process". By it, I did not mean another ikura-shim process, and I did not even mean another another OS process, but I meant it in a more general sense: like as in "distributed process".